### PR TITLE
julia: fix arch conditional

### PR DESCRIPTION
**Pull request by miccal**

### CODE REVIEW

**Comment by jan-xyz** [comment]
nice! Thanks for the quick fix ❤️

File: Casks/julia.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask "julia" do
   arch arm: "aarch64", intel: "x64"
-  midfix = on_arch_conditional arm: arch
```
Comment (jan-xyz) [inline Casks/julia.rb:3]: I think this needs to be `aarch` (double `a`) from the link: https://julialang-s3.julialang.org/bin/mac/aarch64/1.8/julia-1.8.0-macaarch64.dmg

**Comment by adavidzh** [comment]
I wonder if this also fixes what I tried to address in https://github.com/Homebrew/homebrew-cask/pull/130044.

**Comment by miccal** [comment]
Fixed by https://github.com/Homebrew/homebrew-cask/pull/130045

**Pull request closed without merging**